schemas/messaging-system/cfr/cfr.1.schema.json

```diff
@@ -0,0 +1,69 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "anyOf": [
+    {
+      "required": [
+        "message_id"
+      ]
+    },
+    {
+      "required": [
+        "bucket_id"
+      ]
+    }
+  ],
+  "properties": {
+    "addon_version": {
```

Comment:
Is this addon_version required anymore?  I think this is a legacy field from when this was a TestPilot project?

Reply:
Since we could export multiple versions in a single release cycle, I think this is still useful for us to investigate the potential issues.